File: week-01/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int num;
    do
    {
        num = get_int("Height: ");
    }
    while (num <= 0 || num > 8);

    for (int i = 1; i < num + 1; i++)
    {

        for (int j = num - i; j > 0; j--)
        {
            printf(" ");
        }

        for (int l = i; l > 0; l--)
        {
            printf("#");
        }

        printf("  ");

        for (int l = i; l > 0; l--)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: week-06/sentimental-mario-less/mario.py
from cs50 import get_int

height = get_int("Height: ")
while (height > 8 or height < 1):
    height = get_int("Height: ")

for i in range(height):
    print("*" * (height - i))
    height += 1
    i -= 1

print("Starting week 06")